cmds/tr/tr.go

```diff
@@ -0,0 +1,306 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// tr - translate or delete characters
+
+// Synopsis:
+//     tr [OPTION]... SET1 [SET2]
+
+// Description:
+//     Translate, squeeze, and/or delete characters from standard input, writing
+//     to standard output.
+//
+//     -d: delete characters in SET1, do not translate
+//
+// SETs  are  specified  as  strings of characters. Most represent themselves.
+// Interpreted sequences are:
+//     \\        backslash
+//     \a        audible BEL
+//     \b        backspace
+//     \f        form feed
+//     \n        new line
+//     \r        return
+//     \t        horizontal tab
+//     \v        vertical tab
+//     [:alnum:] all letters and digits
+//     [:alpha:] all letters
+//     [:digit:] all digits
+//     [:graph:] all printable characters
+//     [:cntrl:] all control characters
+//     [:lower:] all lower case letters
+//     [:upper:] all upper case letters
+//     [:space:] all whitespaces
+
+package main
+
+import (
+	"bufio"
+	"flag"
```

Comment:
use 
flag "github.com/spf13/flags"

for our flag package

Reply:
imported